api/app/signals/apps/signals/models/attachment.py

```diff
@@ -37,6 +37,12 @@ class Meta:
             models.Index(fields=['is_image']),
             models.Index(fields=['_signal', 'is_image']),
         ]
+        permissions = [
```

Comment:
Please change the names of these permission so that they match all other custom permissions.

All custom permissions that we implement start with `sia_` see the Signal model as an example (sia_read, sia_signal_create_initial, sia_signal_change_status, etc..

This way they are automatically exposed through the permissions endpoint and we are able to set these permission through the FE backoffice. The permission endpoint exposes the permissions starting with `sia_` and as an exception `push_to_sigmax`.

```
queryset = Permission.objects.prefetch_related(
    'content_type',
).filter(Q(codename__istartswith='sia_') | Q(codename='push_to_sigmax'))
```

Reply:
Good point, and done!